eventmesh-protocol-plugin/eventmesh-protocol-grpc/build.gradle

```diff
@@ -24,16 +24,16 @@ repositories {
     mavenCentral()
 }
 
-def grpcVersion = '1.64.0'
+
 def protobufVersion = '3.25.3'
```

Comment:
Looks good! How about adding `protobufVersion` to dependency management as well? In that case, there would need to be a blank line separating the dependencies that use `grpcVersion` from the dependencies that use `protobufVersion`. Of couse, adding two comments is the clearest way if needed.

Also, the whole line should be deleted here, without leaving a blank line.

Reply:
Other than protobufVersion, anything else?

Reply:
Are there more version number variables that haven't been added to dependency management? It looks like there are no more.

Reply:
So, nothing else I think.

Reply:
Excuse me, I haven't seen any changes on `protobufVersion`.

Reply:
Dependency management is only valuable if it is referenced in multiple modules. Managing dependencies in a single module reduces the complexity of the `build.gradle` file in the root directory.

Reply:
That's exactly what I mean. There are still many dependencies in the project that need to be managed.

Reply:
May you give an example? If this dependency does need to be managed, you can continue optimizing it in a subsequent PR.